src/Integration.Vsix.UnitTests/SonarAnalyzer/SonarAnalyzerManagerForBoundTests.cs

```diff
@@ -18,12 +18,12 @@
 namespace SonarLint.VisualStudio.Integration.UnitTests.SonarAnalyzer
 {
     [TestClass]
-    public class SonarAnalyzerDeactivationManagerForBoundTests
+    public class SonarAnalyzerManagerForBoundTests
```

Comment:
You can use partial classes and name the file <Type>Tests.<Focus>.cs in addition to the existing <Type>Tests.cs this will make this clear otherwise it is not consistent.